products/early_access_features/frontend/InstructionsModal.tsx

```diff
@@ -28,10 +28,9 @@ export function InstructionsModal({ onClose, visible, flag }: InstructionsModalP
                         <div>
                             Give your users a{' '}
                             <Link
-                                to={urls.pipelineNodeNew(
-                                    PipelineStage.SiteApp,
-                                    preflight?.region === Region.EU ? 332 : 574
-                                )}
+                                to={urls.pipelineNodeNew(PipelineStage.SiteApp, {
```

Comment:
I know this is not your stuff, but the numbers here look a bit magic.

Reply:
Yeah, absolutely. I dont even know *where* these are defined in the codebase tbh. @PostHog/team-feature-flags in case you wanna make this slightly clearer for future readers